### ISSUE TITLE
BUG: error message -- code working fine though

### ISSUE THREAD

**Issue by fbarfi** [scipy/scipy#15919]
### Describe your issue.

I am getting the error message down below in running MCMC on PyMC on Apple M1. I am posting it here because I see a runtime warning originating in Scipy. I was wondering if v 1.8.0 has addressed this issue. Currently I am using v.1.7.3. The code is running well however, it is just I keep getting this error message and so two of students who have M1 machines too.

here info on my system:
```python
Python implementation: CPython
Python version       : 3.9.10
IPython version      : 8.0.1

Compiler    : Clang 11.1.0 
OS          : Darwin
Release     : 21.4.0
Machine     : arm64
Processor   : arm
CPU cores   : 10
Architecture: 64bit

pymc      : 4.0.0b3
aesara    : 2.4.0
pandas    : 1.4.1
matplotlib: 3.5.1
arviz     : 0.11.4
numpy     : 1.21.5
seaborn   : 0.11.2

```

Thank you for any help or clarification. 

```python

/opt/homebrew/Caskroom/miniforge/base/envs/pymc-dev-py39/lib/python3.9/site-packages/scipy/stats/_continuous_distns.py:624: RuntimeWarning: overflow encountered in _beta_ppf
  return _boost._beta_ppf(q, a, b)

```

### Reproducing Code Example

```python

```

### Error message

```shell
/opt/homebrew/Caskroom/miniforge/base/envs/pymc-dev-py39/lib/python3.9/site-packages/scipy/stats/_continuous_distns.py:624: RuntimeWarning: overflow encountered in _beta_ppf
  return _boost._beta_ppf(q, a, b)
```

### SciPy/NumPy/Python version information

1.7.3 1.21.5 sys.version_info(major=3, minor=9, micro=10, releaselevel='final', serial=0)

**Comment by mdhaber** [comment]
Thanks, but this is a duplicate of gh-14901, so I'll close this one.

**Comment by Yuan-Meng** [comment]
> 

Hi there! I encountered the same problem and read the issue you referred to. However, it says the problem was fixed in 1.7.2 so it doesn't really address why we still see it in 1.7.3?
